addons/event/models/event_event.py

```diff
@@ -644,6 +645,17 @@ def mail_attendees(self, template_id, force_send=False, filter_func=lambda self:
             for attendee in event.registration_ids.filtered(filter_func):
                 self.env['mail.template'].browse(template_id).send_mail(attendee.id, force_send=force_send)
 
+    def _get_external_description(self):
+        """ Description of the event shortened to maximum 2000 characters.
+        Meant to be used for external content (ics/icalc/Gcal).
+
```

Comment:
Would mention that we actually shorten it to 1900 to leave some space for addition by sub-modules, such as the even link.